### ISSUE TITLE
Add GitHub releases

### ISSUE THREAD

**Issue by baimafeima** [maxnd/mynotex#1]
Hi Massimo, I am very excited to hear about your software. Could you add releases here on GitHub? Google is blocked in China and GitHub is pretty much the only reliable way to download tarballs.

**Comment by maxnd** [comment]
You may try https://www.ctan.org/pkg/bibfilex.

Massimo

**Comment by maxnd** [comment]
OK, I'll do with the next upcoming version.

Massimo

**Comment by maxnd** [comment]
Done, the packages of the 1.4.1 version are in the folder Packages.

**Comment by baimafeima** [comment]
Is it possible to release versioned tarballs?

**Comment by maxnd** [comment]
No, I don't keep them even in the website of the software. Anyway, you may download from this repository any version of the source code and compile it with Lazarus.

**Comment by baimafeima** [comment]
Tagging your releases here on GitHub has several advantages. You could also make the changelog visible and thereby increase transparency for potential users and distribution maintainers. New version updates can then be tracked more easily. Something like https://skywinder.github.io/github-changelog-generator/ could be used to generate the changelog based on merged PRs and closed issues since the last release tag.
